cmd/kat-server/services/grpc-auth-v3.go

```diff
@@ -99,6 +99,16 @@ func (g *GRPCAUTHV3) Check(ctx context.Context, r *pb.CheckRequest) (*pb.CheckRe
 		rheader["body"] = rbody
 	}
 
+	rContextExtensions := r.GetAttributes().GetContextExtensions()
+	if rContextExtensions != nil {
+		val, err := json.Marshal(rContextExtensions)
+		if err != nil {
+			val = []byte(fmt.Sprintf("Error: %v", err))
+		}
+
+		rs.AddHeader(false, "x-request-context-extensions", string(val))
+	}
+
```

Comment:
This is in the `upgrade to envoy 1.22.2` commit; doesn't it belong in the `removes support for envoy v2 transport protocol` commit?